google-datacatalog-rdbms-connector/src/google/datacatalog_connectors/rdbms/prepare/sql_objects/sql_objects_datacatalog_entry_factory.py

```diff
@@ -20,7 +20,7 @@
 from google.datacatalog_connectors.commons.prepare.base_entry_factory import \
```

Comment:
WDYT about leveraging the present PR to fix the class import and reorder all imports as well?